src/drivers/localstorage.js

```diff
@@ -39,6 +39,10 @@
 
         dbInfo.keyPrefix = dbInfo.name + '/';
 
+        if (dbInfo.storeName !== self._defaultConfig.storeName) {
+            dbInfo.keyPrefix += dbInfo.storeName + '/';
+        }
```

Comment:
No issue in particular AFAIK, but necessary to fulfill the new test case of this PR.

In fact, it's surprising the support of `storeName` hasn't been implemented before. At least in the same time than the two other drivers. In any case, I didn't see anything about that in the localStorage driver.